PROJECT

```diff
@@ -0,0 +1,3 @@
+version: "2"
```

Comment:
We'll be fast following to change this to `3` once we get https://github.com/kubernetes-sigs/kubebuilder/milestone/21 finished. We might even start this with the `3-alpha`.